test/built-ins/String/prototype/matchAll/cstm-matcher-get-err.js

```diff
@@ -1,26 +0,0 @@
-// Copyright (C) 2018 Jordan Harband. All rights reserved.
```

Comment:
This test is now:
  - [regexp-get-matchAll-throws](https://github.com/peterwmwong/test262/blob/matchall-pww/test/built-ins/String/prototype/matchAll/regexp-get-matchAll-throws.js)
  - [regexp-prototype-get-matchAll-throws](https://github.com/peterwmwong/test262/blob/matchall-pww/test/built-ins/String/prototype/matchAll/regexp-prototype-get-matchAll-throws.js)